src/components/Banner/Banner.mdx

```diff
@@ -0,0 +1,236 @@
+
+### Icons
+
+Icons rendered in the `Banner` component are decorative by default which
+means that they are hidden to screen readers. If `isDismissable` is true, the
+"X" close icon is wrapped inside a button with an appropriate `aria-label`
+attribute.
+
+### Language Patterns
+
+When the `Banner` component is rendered, the text within the component
+should always include wording that aligns with the type of notitication that is
+being delivered. For example, if the Banner is a warning or error, the
+heading or body copy within the Banner should use the words `"warning"` or
+`"error"` and provide a clear description of the existing or potential isssue.
+
+### Dynamic Banners
+
+If dynamically generated `Banners` are required, a container into which the
+`Banner` will be injected should be included in the DOM. The container
+element should have `aria-live="polite"` applied. The container element should
+always be rendered in the DOM regardless of the presence of the `Banner`
+component. This DOM structure will allow the newly generated content to be
+recognized and announced by assistive technology.
+
+<Source
+  code={`
+<div aria-live="polite">
+  <Banner {...props} />
+</div>
+`}
+  language="tsx"
+/>
+
+### Resources
+
+- [MDN Aria: complementary role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/complementary_role)
+- [Deque Creating Accessible SVGs](https://www.deque.com/blog/creating-accessible-svgs/)
+- [CSS Tricks Accessible SVG Icons](https://css-tricks.com/accessible-svg-icons/)
+- [Reservoir Accessibility Guide: Dynamic Content](..?path=/docs/accessibility-guide-dynamic-content--docs)
+
+## Variants
+
+The `type` prop can be used to render different variants of `Banner`. Each
+variant has a different color and iconography.
+
+It is recommended to align the usage of the Banner component to the semantic
+meaning of one of the component’s variants. When using the Banner component,
+the implementation should not rely on the color alone to convey meaning, as
+color is not a universal experience for everyone. The meaning of the heading
+and body copy should align with the general meaning associated with the semantic
+colors used in a specific variant.
+
+**IMPORTANT:** The text color for links within the content will NOT use the
+DS standard link color. Instead, the color will match the color of the content.
+Additionally, links will always be underlined.
+
+**IMPORTANT:** For the "negative" variant only, the heading and text color will
+both be set to the same color, `ui.error.primary` and `dark.ui.error.primary`
+for dark mode.
+
+<Canvas of={BannerStories.TypeVariants} />
+
+## Banner Heading
+
+The `Banner` component can be rendered with and without a heading. By
+default, when a title is passed to the `heading` prop, the
+`Banner` component will render an `h2` element sized as a `heading6`
+heading. If this causes accessibility issues in your app, you can pass in a
+custom `Heading` element.
```

Comment:
Same here.  This would be cleared if changed to 
```
If this causes accessibility issues in your app, you can set a custom heading by 
passing in a DS `Heading` component.
```